.changeset/tiny-facts-stay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@terrazzo/parser": minor
```

Comment:
```suggestion
"@terrazzo/parser": patch
```

For 0.x I’ve been using minor versions to mark breaking changes. Plus this is a bugfix anyway (I think). I believe inheritance should have worked